sample_rules/simple/simple0.yaml

```diff
@@ -0,0 +1,7 @@
+# YAML
+# Goal: Turn on back light when in reverse
+
+condition:
+ - state.transmission_gear == 'reverse'
```

Comment:
So this is another weird issue that I found that I had to change the approach. The way python works is yes you can add it to the instance of a class as state.transmission.gear and it will be addressable but if you try to say state.transmission.gear it will say there is no class transmission. So changing how it works is the only valid way of doing it here. So it's either substitution or changing how the yaml is designed instead.